src/app/src/main/java/com/example/bookmark/MyBookDetailsActivity.java

```diff
@@ -284,6 +297,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     @Override
     protected void onResume() {
         super.onResume();
+        loadImage();
```

Comment:
Does this work? We load an image and then fetch the book?

Reply:
Why not fetch book and then load an image?

Reply:
Yeah actually it makes more sense to fetch a book before trying to fetch an image. This was causing a bug initially.

Reply:
whoops

Reply:
also make sure to keep the book != null check because if a book is deleted it becomes null